includes/compat/class-log.php

```diff
@@ -331,8 +340,22 @@ public function file_download_log_update_post_meta( $check, $object_id, $meta_ke
 					) );
 				}
 				break;
+
+			case '_edd_log_customer_id':
```

Comment:
I don't think this is needed? Since you already added a `case '_edd_log_customer_id':` on line 324